### PR TITLE
I haven't tested this at all 

### PR DESCRIPTION
**Pull request by alicolver**
Plz fix as needed!

### CODE REVIEW

File: src/blueprints/authentication.py [hunk 1]
```diff
@@ -151,90 +151,91 @@ def isAdmin():
...
+                'message': 'expired'
+            })
+
+        if not pbkdf2_sha256.verify(one_time_password, passwordResetInfo.one_time_password):
+            return jsonify({
+                'success': False,
+                'message': 'otp'
+            })
+
+        passwordResetInfo.has_reset = True
```
Comment (lukeely) [inline src/blueprints/authentication.py:234]: I think we need a session.commit() after this

Comment (alicolver) [reply src/blueprints/authentication.py:234]: Yeah good spot

**Comment by lukeely** [comment]
Just need to fix the HTML emails and this can go in

**Review by lukeely — APPROVED** [review]
Tested locally, works well

**Pull request merged**